config/jobs/kubernetes/sig-release/release-branch-jobs/1.11.yaml

```diff
@@ -599,3 +627,48 @@ periodics:
           resources:
             requests:
               cpu: 4
+  # This is a sig-release-<stable3>-blocking job.
+  - cron: "0 8-20/12 * * *" # every six hours, disjoint with ci-kubernetes-e2e-gci-gce-scalability-stable1 and ci-kubernetes-e2e-gci-gce-scalability-stable2
+    name: ci-kubernetes-e2e-gci-gce-scalability-stable3
+    tags:
+      - "perfDashPrefix: gce-100Nodes-stable3"
+      - "perfDashJobType: performance"
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-e2e-scalability-common: "true"
+    annotations:
+      testgrid-dashboards: sig-release-1.11-blocking, sig-release-1.11-all, sig-scalability-gce, google-gce
```

Comment:
I don't know the annotation values supported map. Something like below is less verbose and well namespaced.
```
"annoations":
    testgrid: {
    dashboards: blah
}
```

Reply:
They are only `key: value` (https://github.com/kubernetes/test-infra/blob/master/prow/config/jobs.go#L108)

I think it would be a mistake to jam machine-parseable yaml/json in there as I've seen other things do with Kubernetes resource annotations

Namespacing to the package name of the thing that parses these might be discoverable, but I'm fine with that as a followup